### ISSUE TITLE
Android app can not load interactive rewarded ad from Cocos Engine

### ISSUE THREAD

**Issue by ilker-aktuna** [cocos/cocos-engine#16765]
### Cocos Creator version

3.5

### System information

Android app

### Issue description

I am a mobile app developer. In one of my apps, I use Admob rewarded ad. 
Sometimes when Admob tries to load an ad, it receives a Cocos Engine ad but can not display it.
I see the following logs:

### Relevant error log output

> 
> 2024-03-03 20:09:35.061 30477-30477 Ads                     com.example.android.app                I  JS: Create unpacker %s for %s 0c649e7a9 02delMVqdBD70a/HSD99FK (https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html:233)
> 2024-03-03 20:09:35.061 30477-30477 chromium                com.example.android.app                I  [INFO:CONSOLE(233)] "Create unpacker %s for %s 0c649e7a9 02delMVqdBD70a/HSD99FK", source: https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html (233)
> 2024-03-03 20:09:35.071 30477-30477 Ads                     com.example.android.app                I  JS: Cocos Creator v2.1.2 (https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html:233)
> 2024-03-03 20:09:35.071 30477-30477 chromium                com.example.android.app                I  [INFO:CONSOLE(233)] "Cocos Creator v2.1.2", source: https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html (233)
> 2024-03-03 20:09:35.072 30477-30477 Ads                     com.example.android.app                E  JS: Permissions policy violation: fullscreen is not allowed in this document. (https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html:233)
> 2024-03-03 20:09:35.072 30477-30477 chromium                com.example.android.app                I  [INFO:CONSOLE(233)] "Permissions policy violation: fullscreen is not allowed in this document.", source: https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html (233)
> 2024-03-03 20:09:35.073 30477-30477 Ads                     com.example.android.app                I  JS: Create unpacker %s for %s 0be4c3c23 a5Bjm7hetFg6GbLnEdU0We (https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html:233)
> 2024-03-03 20:09:35.073 30477-30477 chromium                com.example.android.app                I  [INFO:CONSOLE(233)] "Create unpacker %s for %s 0be4c3c23 a5Bjm7hetFg6GbLnEdU0We", source: https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html (233)
> 2024-03-03 20:09:35.074 30477-30477 Ads                     com.example.android.app                E  JS: Uncaught (in promise) TypeError: Disallowed by permissions policy (https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html:233)
> 2024-03-03 20:09:35.074 30477-30477 chromium                com.example.android.app                I  [INFO:CONSOLE(233)] "Uncaught (in promise) TypeError: Disallowed by permissions policy", source: https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html (233)
> 2024-03-03 20:09:35.084 30477-30477 Ads                     com.example.android.app                W  JS: Warning 5302, please go to https://github.com/cocos-creator/engine/blob/master/EngineErrorMap.md#5302 to see details. Arguments: cc.Layout (https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html:233)
> 2024-03-03 20:09:35.084 30477-30477 chromium                com.example.android.app                I  [INFO:CONSOLE(233)] "Warning 5302, please go to https://github.com/cocos-creator/engine/blob/master/EngineErrorMap.md#5302 to see details. Arguments: cc.Layout", source: https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html (233)
> 2024-03-03 20:09:35.087 30477-30477 Ads                     com.example.android.app                W  JS: Warning 5302, please go to https://github.com/cocos-creator/engine/blob/master/EngineErrorMap.md#5302 to see details. Arguments: cc.Layout (https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html:233)
> 2024-03-03 20:09:35.087 30477-30477 chromium                com.example.android.app                I  [INFO:CONSOLE(233)] "Warning 5302, please go to https://github.com/cocos-creator/engine/blob/master/EngineErrorMap.md#5302 to see details. Arguments: cc.Layout", source: https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html (233)
> 2024-03-03 20:09:35.088 30477-30477 Ads                     com.example.android.app                W  JS: Warning 5302, please go to https://github.com/cocos-creator/engine/blob/master/EngineErrorMap.md#5302 to see details. Arguments: cc.Layout (https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html:233)
> 2024-03-03 20:09:35.088 30477-30477 chromium                com.example.android.app                I  [INFO:CONSOLE(233)] "Warning 5302, please go to https://github.com/cocos-creator/engine/blob/master/EngineErrorMap.md#5302 to see details. Arguments: cc.Layout", source: https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html (233)
> 2024-03-03 20:09:35.089 30477-30477 Ads                     com.example.android.app                W  JS: Warning 5302, please go to https://github.com/cocos-creator/engine/blob/master/EngineErrorMap.md#5302 to see details. Arguments: cc.Layout (https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html:233)
> 2024-03-03 20:09:35.089 30477-30477 chromium                com.example.android.app                I  [INFO:CONSOLE(233)] "Warning 5302, please go to https://github.com/cocos-creator/engine/blob/master/EngineErrorMap.md#5302 to see details. Arguments: cc.Layout", source: https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html (233)
> 2024-03-03 20:09:35.090 30477-30477 Ads                     com.example.android.app                W  JS: Warning 5302, please go to https://github.com/cocos-creator/engine/blob/master/EngineErrorMap.md#5302 to see details. Arguments: cc.Layout (https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html:233)
> 2024-03-03 20:09:35.090 30477-30477 chromium                com.example.android.app                I  [INFO:CONSOLE(233)] "Warning 5302, please go to https://github.com/cocos-creator/engine/blob/master/EngineErrorMap.md#5302 to see details. Arguments: cc.Layout", source: https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html (233)
> 2024-03-03 20:09:35.090 30477-30477 Ads                     com.example.android.app                W  JS: Warning 5302, please go to https://github.com/cocos-creator/engine/blob/master/EngineErrorMap.md#5302 to see details. Arguments: cc.Layout (https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html:233)
> 2024-03-03 20:09:35.090 30477-30477 chromium                com.example.android.app                I  [INFO:CONSOLE(233)] "Warning 5302, please go to https://github.com/cocos-creator/engine/blob/master/EngineErrorMap.md#5302 to see details. Arguments: cc.Layout", source: https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html (233)
> 2024-03-03 20:09:35.090 30477-30477 Ads                     com.example.android.app                W  JS: Warning 5302, please go to https://github.com/cocos-creator/engine/blob/master/EngineErrorMap.md#5302 to see details. Arguments: cc.Layout (https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html:233)
> 2024-03-03 20:09:35.090 30477-30477 chromium                com.example.android.app                I  [INFO:CONSOLE(233)] "Warning 5302, please go to https://github.com/cocos-creator/engine/blob/master/EngineErrorMap.md#5302 to see details. Arguments: cc.Layout", source: https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html (233)
> 2024-03-03 20:09:35.218 30477-30477 Ads                     com.example.android.app                I  JS: LoadScene a5Bjm7hetFg6GbLnEdU0We: 144.85205078125 ms (https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html:233)
> 2024-03-03 20:09:35.218 30477-30477 chromium                com.example.android.app                I  [INFO:CONSOLE(233)] "LoadScene a5Bjm7hetFg6GbLnEdU0We: 144.85205078125 ms", source: https://tpc.googlesyndication.com/sadbundle/$csp%3Der-p$/8703987913591843304/index.html (233)
> 

### Steps to reproduce

not easy. Use a rewarded ad in an Android app and then wait for a Cocos-Engine created interactive ad :)

**Comment by minggo** [comment]
Did you create ad by using cocos creator?

**Comment by ilker-aktuna** [comment]
no. Actually I only have Admob integrated on my app. Admob sends an ad created by cocos creator.
I don't have control over it.

**Comment by minggo** [comment]
Sorry, we don't know how to help you. It seems it is a permission error.

**Comment by ilker-aktuna** [comment]
yes but this seems to be a general issue. It must be affecting all apps who receive ads created by Cocos Engine.
And these apps do not show these ads. 
I could not find any permission which is related with that. If ads from Cocos Engine should be visible on Android ads, maybe you have to change something.
